client/src/components/TabbedContent.js

```diff
@@ -15,23 +15,24 @@ export class TabbedControls extends React.Component {
           {_.map(tab_options, 
             ({key, label, is_open, is_disabled}) => (
               <li
-                className = { classNames({
+                className={ classNames({
                   'tabbed-controls__label': true,
                   'tabbed-controls__label--active': !!is_open,
                   'tabbed-controls__label--disabled': !!is_disabled,
                 }) }
-                id = {key + "_tab"}
-                key = {key + "_tab"}
-                onClick = { () => !is_disabled && tab_callback(key) }
+                id={key + "_tab"}
+                key={key + "_tab"}
+                onClick={ () => !is_disabled && tab_callback(key) }
                 title={is_disabled ? disabled_message : ''}
               > 
                 <span
-                  tabIndex = {0} 
-                  className = "tabbed-controls__label-text"
-                  role = "button"
-                  aria-pressed = { is_open }
-                  onClick = { () => !is_disabled && tab_callback(key) }
-                  onKeyDown = { (e) => !is_disabled && (e.keyCode===13 || e.keyCode===32) && tab_callback(key) }
+                  tabIndex={0}
+                  className="tabbed-controls__label-text"
+                  role="button"
+                  aria-pressed={ is_open }
+                  aria-disabled={ is_disabled }
```

Comment:
Hard to see because of all the code style tweaks, but the only a11y changes I made to `TabbedContent` were properly flagging disabled state to screen readers here, and removing the `aria-hidden` below.

Reply:
Accessible rich internet applications: for improving accessibility (visibility for people using screen readers)
👍